xbmc/input/keymaps/remote/IRTranslator.cpp

```diff
@@ -318,3 +329,29 @@ uint32_t CIRTranslator::TranslateUniversalRemoteString(const std::string& szButt
 
   return buttonCode;
 }
+
+uint32_t CIRTranslator::ApplyModifiersToButton(const tinyxml2::XMLElement* pButton,
+                                               uint32_t iButtonCode)
+{
+  // Process the longpress modifier
+  std::string strMod;
+  if (pButton->Attribute("mod"))
+  {
+    strMod = pButton->Attribute("mod");
+    StringUtils::ToLower(strMod);
+
+    std::vector<std::string> modArray = StringUtils::Split(strMod, ",");
+    for (auto substr : modArray)
+    {
+      StringUtils::Trim(substr);
+
+      if (substr == "longpress")
+        iButtonCode |= CKey::MODIFIER_LONG;
+      else
+        CLog::Log(LOGERROR, "Remote Translator: Unknown key modifier %s in %s", substr.c_str(),
```

Comment:
Please update to the current log standards using `{}`.

For more information please see the [libfmt documentation](https://fmt.dev/latest/syntax.html)